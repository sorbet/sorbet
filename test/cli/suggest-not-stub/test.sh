#!/bin/sh
#!/bin/bash
main/sorbet --silence-dev-message test/cli/suggest-not-stub/suggest-not-stub.rb 2>&1
