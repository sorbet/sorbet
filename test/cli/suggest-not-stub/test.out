test/cli/suggest-not-stub/suggest-not-stub.rb:3: Unable to resolve constant `B` https://srb.help/5002
     3 |  B
          ^

test/cli/suggest-not-stub/suggest-not-stub.rb:6: Unable to resolve constant `B` https://srb.help/5002
     6 |B
        ^
  Did you mean `T`? Use `-a` to autocorrect
    test/cli/suggest-not-stub/suggest-not-stub.rb:6: Replace with `T`
     6 |B
        ^
    https://github.com/sorbet/sorbet/tree/master/rbi/sorbet/t.rbi#LCENSORED: `T` defined here
    16 |module T
        ^^^^^^^^
  Did you mean `A`? Use `-a` to autocorrect
    test/cli/suggest-not-stub/suggest-not-stub.rb:6: Replace with `A`
     6 |B
        ^
    test/cli/suggest-not-stub/suggest-not-stub.rb:2: `A` defined here
     2 |class A
        ^^^^^^^
Errors: 2
